sct.py

```diff
@@ -342,6 +342,9 @@ def list_ami_branch(region, version):
     def get_tags(ami):
         return {i['Key']: i['Value'] for i in ami.tags}
 
+    if ":" not in version:
+        version += ":all"
```

Comment:
shouldn't it be `:latest` instead `:all`

Reply:
Default value for `version` is `branch-3.1:all`. It's `hydra list-ami-branch` command. I think, it's OK to list them all. Just to not raise exception in such trivial case.

Reply:
i was just thinking in the case where user entered a branch, so we could already set it to the latest, no?

Reply:
the default should be `latest` not all

Reply:
Example of output:

```
$ hydra list-ami-branch branch-4.2
...
Looking for AMI match [branch-4.2:all]
+---------------------------------------------------------------------------------------------------------------------+
|                                              Scylla AMI branch versions                                             |
+------------------------------------------+-----------------------+--------------------------+---------+-------------+
| Name                                     | ImageId               | CreationDate             | BuildId | Test Status |
+------------------------------------------+-----------------------+--------------------------+---------+-------------+
| Scylla-branch-4.2-nightly-20201014111324 | ami-................. | 2020-10-14T08:13:25.000Z | 89      | Unknown     |
| Scylla-branch-4.2-nightly-20201013110722 | ami-................. | 2020-10-13T08:07:23.000Z | 88      | Unknown     |
| Scylla-branch-4.2-nightly-20201012171152 | ami-................. | 2020-10-12T14:11:53.000Z | 87      | Unknown     |
| Scylla-branch-4.2-nightly-20201012105138 | ami-................. | 2020-10-12T07:51:39.000Z | 86      | Unknown     |
| Scylla-branch-4.2-nightly-20201009111047 | ami-................. | 2020-10-09T08:10:48.000Z | 85      | Unknown     |
| Scylla-branch-4.2-nightly-20201008110603 | ami-................. | 2020-10-08T08:06:04.000Z | 84      | Unknown     |
...
```

They sorted. The first one is the latest.

Reply:
but better to have only the last one, as user inputs only 1 AMI version?

Reply:
Should we change default value for `version` argument from `branch-3.1:all` to `master:latest` too?

Reply:
if one reads the command `hydra list-ami-branch` and you don't know how it works you read it as a text:
hydra list ami branch, i.e. list me everything that is related to branch. Also default is "all" so let's don't add hidden logic, if somebody needs the latest, he should mention it explicitly, or we should change the default. 
Let's merge this as is. If somebody still thinks different let's raise this in the QA meeting